pages/common/history.md

```diff
@@ -0,0 +1,11 @@
+# history
+
+> Command Line history.
+
+- Display the history list with line number
+
+`history`
+
+- Clear the history list
+
+`history -c`
```

Comment:
Interesting.

You are probably using Zsh/Oh-My-Zsh. `history` is an internal command of shell. In `bash` and `sh` it has `-c` key, in `zsh` it does not have it.

I think that we should keep `-c`, because `sh` and `bash` are still more useful. Maybe I should add note

~~~Markdown
- Clear the history list (in `sh` and `bash`)
~~~